docs/api/ref/DesktopAgent.md

```diff
@@ -383,8 +389,18 @@ if (fdc3.getInfo && versionIsAtLeast(await fdc3.getInfo(), "1.2")) {
 }
 ```
 
+The `ImplementationMetadata` object returned also includes the metadata for the calling application, according to the desktop agent. This allows the application to retrieve its own `appId`, `instanceId` and other details, e.g.:
```

Comment:
desktop agent is capitalized in the section above. I forget if we always want to capitalize? or if Desktop Agent refers to the specification and desktop agent refers to an instance?

Reply:
I'm not sure TBH, its probably easier to stick to capitalisation...

```suggestion
The `ImplementationMetadata` object returned also includes the metadata for the calling application, according to the Desktop Agent. This allows the application to retrieve its own `appId`, `instanceId` and other details, e.g.:
```

Reply:
Capitalisation is all over the place in this doc and the spec, will fix the ones I can see in this PR to use caps